### PR TITLE
Background additions

### PR DESCRIPTION
**Pull request by davidkaneda**
- Add background-gradient mixin, along with accompanying functions for matte, bevel, glossy, recessed
- Add a carbon fiber pattern
- Deprecate ui-glossy
- Tests included

### CODE REVIEW

File: stylesheets/recipes/_background.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
-@import "recipes/background/blueprint-grid";
 @import "recipes/background/noise";
+@import "recipes/background/gradients";
+@import "recipes/background/blueprint-grid";
```
Comment (MoOx) [inline stylesheets/recipes/_background.scss:3]: Why do you move this one ? Since now, I've used alphabetical order.
What can you suggest ?

Comment (davidkaneda) [inline stylesheets/recipes/_background.scss:3]: Ah, sorry, didn't realize it was alphabetical. I just wanted to keep "patterns" together, whereas noise and gradients are separate.

File: stylesheets/recipes/background/_carbon-fiber.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Carbon Fiber background pattern
+ * 
+ * @link http://lea.verou.me/css3patterns/
+ *
+ * @author Lea Verou http://lea.verou.me/ for the original pattern
+ * @author David Kaneda http://www.davidkaneda.com/ for the Sass mixin
+ *
+ * WebKit only for now.
```
Comment (MoOx) [inline stylesheets/recipes/background/_carbon-fiber.scss]: Why ?

File: stylesheets/recipes/background/_carbon-fiber.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+@mixin background-carbon-fiber(
+    $bgcolor: $base-color,
+    $dotcolor: darken($bgcolor, 10),
+    $highlightcolor: rgba(#fff, .1),
+    $spacing-size: 20px,
+    $dotsize: 20%
+) {
+
+    background: 
+        -webkit-radial-gradient($dotcolor $dotsize, rgba(#fff, 0) $dotsize + 1) 0 0,
```
Comment (MoOx) [inline stylesheets/recipes/background/_carbon-fiber.scss]: Why don't use compass function radial-gradient() ?

File: stylesheets/recipes/background/_carbon-fiber.scss (hunk 2 above)
Comment (davidkaneda) [inline stylesheets/recipes/background/_carbon-fiber.scss]: Not entirely sure, ha, it's just how it was in Lea's example. I'll give it a shot.

File: stylesheets/recipes/background/_carbon-fiber.scss (hunk 3 above)
Comment (davidkaneda) [inline stylesheets/recipes/background/_carbon-fiber.scss]: Will do the switch now-

File: stylesheets/recipes/background/_carbon-fiber.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+@import "compass/css3/images";
+
+@mixin background-carbon-fiber(
+    $bgcolor: $base-color,
+    $dotcolor: darken($bgcolor, 10),
+    $highlightcolor: rgba(#fff, .1),
+    $spacing-size: 20px,
+    $dotsize: 20%
+) {
```
Comment (MoOx) [inline stylesheets/recipes/background/_carbon-fiber.scss:21]: I like to put the { on the next line, to create more readable file. Are you against this ?

Comment (davidkaneda) [inline stylesheets/recipes/background/_carbon-fiber.scss:21]: Ha, yeah, I was meaning to ask you. I feel like it's more of a standard to not, but if you feel strongly about it, I'll switch. I used to do that with PHP, but most people keep the { on the same line in CSS...

File: stylesheets/recipes/background/_carbon-fiber.scss (hunk 3 above)
Comment (davidkaneda) [inline stylesheets/recipes/background/_carbon-fiber.scss]: Fixed. :)

File: stylesheets/recipes/background/_carbon-fiber.scss (hunk 4 above)
Comment (MoOx) [inline stylesheets/recipes/background/_carbon-fiber.scss:21]: Yes, most people does that. But I think it's a bad habit :)

File: stylesheets/recipes/ui/_glossy.scss [hunk 5]
```diff
@@ -7,6 +7,9 @@
  */
 @mixin ui-glossy($color: #000, $opacity: .6, $border-width: .1em, $border-color-top: rgba(255,255,255,.5), $border-color-bottom: rgba(85,85,85,.6))
 {
+
+    @warn '@mixin ui-glossy has been deprecated: Use background-gradient mixin/functions instead.';
```
Comment (MoOx) [inline stylesheets/recipes/ui/_glossy.scss:11]: Please add something like a "@todo remove this in futur release 0.4". And in the warning too :)

**Comment by MoOx** [comment]
Great job with that. We'll fix tiny stuffs later ,)

**Pull request merged**